### PR TITLE
Wrote Bootstrap Buttons page.

### PR DESCRIPTION
**Pull request by nsuchy**
This pull request is to add the Bootstrap Buttons page that I wrote.

### CODE REVIEW

File: src/pages/articles/bootstrap/bootstrap-buttons/index.md [hunk 1]
```diff
@@ -2,14 +2,35 @@
...
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+`<button type="button" class="btn btn-primary">Primary</button>`
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+#### Bootstrap Button Class List:
+This is a list of the CSS classes that bootstrap provides for buttons.
+
+`.btn` This is bootstrap's basic button.
+
```
Comment (Bouncey) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: Consider an actual example button, as above, for each of these classes

File: src/pages/articles/bootstrap/bootstrap-buttons/index.md [hunk 2]
```diff
@@ -2,14 +2,35 @@
...
 
-This is a stub. [Help our community expand it](https://github.com/freecodecamp/guides/tree/master/src/pages/articles/bootstrap/bootstrap-buttons/index.md).
+#### How To Use:
+To use bootstrap buttons you follow the same steps that you would to create a button in HTML except you also apply the applicable CSS class to the button. A code example has been provided below.
 
-[This quick style guide will help ensure your pull request gets accepted](https://github.com/freecodecamp/guides/blob/master/README.md).
+**Code Example:**
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+`<button type="button" class="btn btn-primary">Primary</button>`
```
Comment (Bouncey) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: You will have access to bootstrap styles here. Including an actual HTML button with the primary class will help this article.

There is no special syntax for including HTML in markdown, you can write HTML and markdown in the same file :+1:

Comment (nsuchy) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: I didn't realize I could use plain html, I'll add the buttons shortly.

File: src/pages/articles/bootstrap/bootstrap-buttons/index.md (hunk 1 above)
Comment (nsuchy) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: See comment above.

**Comment by nsuchy** [comment]
OK - So everything is squashed into 1 commit now, I had to fix an issue with merge conflicts which should be fixed. Please review the code and let me know if other changes are requested.

File: src/pages/articles/bootstrap/bootstrap-buttons/index.md [hunk 3]
```diff
@@ -2,14 +2,36 @@
...
 
-This is a stub. [Help our community expand it](https://github.com/freecodecamp/guides/tree/master/src/pages/articles/bootstrap/bootstrap-buttons/index.md).
+#### How To Use:
+To use bootstrap buttons you follow the same steps that you would to create a button in HTML except you also apply the applicable CSS class to the button. A code example has been provided below.
 
-[This quick style guide will help ensure your pull request gets accepted](https://github.com/freecodecamp/guides/blob/master/README.md).
+**Code Example:**
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+<button type="button" class="btn btn-primary">Primary</button>
```
Comment (Bouncey) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: Can we have the button element underneath the code snippet please?

File: src/pages/articles/bootstrap/bootstrap-buttons/index.md [hunk 4]
```diff
@@ -2,14 +2,36 @@
...
+`<button type="button" class="btn btn-primary">Primary</button>`
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+#### Bootstrap Button Class List:
+This is a list of the CSS classes that bootstrap provides for buttons.
+
+`.btn` <button type="button" class="btn">Basic</button> This is bootstrap's basic button.
```
Comment (Bouncey) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: Can we have the button element on a new line after `This is bootstrap's ${buttonType} button`

And the same for each example :+1:

File: src/pages/articles/bootstrap/bootstrap-buttons/index.md (hunk 3 above)
Comment (nsuchy) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: Fixed, commit amended.

File: src/pages/articles/bootstrap/bootstrap-buttons/index.md (hunk 4 above)
Comment (nsuchy) [inline src/pages/articles/bootstrap/bootstrap-buttons/index.md]: Fixed, commit amended.

**Comment by nsuchy** [comment]
The requested changes were amended into the commit.

**Comment by Bouncey** [comment]
## LGTM 👍 

Thanks for contributing :sparkles:

**Comment by nsuchy** [comment]
@Bouncey You should probably remove the changes requested label now that it's been merged.

**Pull request merged**